.github/workflows/release.yml

```diff
@@ -0,0 +1,63 @@
+name: release
+on:
+  push:
+    tags:
+      - 'v*'
+    branches:
+      - main
+      - dev-*
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build-push-linux-multi:
+    name: Build & push linux/amd64 and linux/arm64
+    needs: [ build-binaries ]
+    runs-on: ubuntu-latest
+    if: github.repository == 'numaproj-labs/numaplane'
+    strategy:
+      matrix:
+        target: [ numaplane ]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Registry Login
+        uses: docker/login-action@v2
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAYIO_USERNAME }}
+          password: ${{ secrets.QUAYIO_PASSWORD }}
+
+      - name: set Version
+        id: version
+        run: |
+          tag=$(basename $GITHUB_REF)
+          if [ $tag = "main" ]; then
+            tag="latest"
+          fi
+          echo "VERSION=$tag" >> $GITHUB_OUTPUT
+      - name: Container build and push with arm64/amd64
+        run: make image docker-push IMAGE_FULL_PATH=${{ secrets.QUAYIO_ORG }}/numaplane-controller:${{ steps.version.outputs.VERSION }} CONTAINER_TOOL=docker
+
+  Release:
+    runs-on: ubuntu-latest
+    if: github.repository == 'numaproj-labs/numaplane'
+    needs: [ build-push-linux-multi ]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Registry Login
+        uses: docker/login-action@v2
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAYIO_USERNAME }}
+          password: ${{ secrets.QUAYIO_PASSWORD }}
+      - name: Release binaries
```

Comment:
```suggestion
      - name: Create Release
```

Reply:
After we merge this, can you try the following to verify what happens when we create a Tag?:
```
git fetch
git checkout -b release-0.2 origin/release-0.2 # checkout the release-0.2 branch
export VERSION=v0.2.2
make prepare-release 
make release  # follow any of the things it says to do
```

Reply:
This is what happen after merging and verifying the steps above. Got an error on step `make prepare-release` @juliev0  

<img width="902" alt="Screenshot 2024-05-14 at 10 33 23 AM" src="https://github.com/numaproj-labs/numaplane/assets/53889037/fbeb7fc5-a656-4e8e-bd9e-99f196af4afc">

Reply:
after solving that i completed the process. Only difference is i did `git push origin` instead of `git push {your-remote}` 
<img width="1000" alt="Screenshot 2024-05-14 at 11 56 30 AM" src="https://github.com/numaproj-labs/numaplane/assets/53889037/7961746e-4b13-409a-9c14-3338c4e34df4">